src/pages/index.astro

```diff
@@ -0,0 +1,515 @@
+      },
+      {
+        icon: 'tabler:number-2',
+        title: 'See all the information from your photo',
+      },
+      {
+        icon: 'tabler:number-3',
+        title: 'Batch selection with Desktop-like multi-select',
+      },
+      {
+        icon: 'tabler:number-4',
+        title: 'Share your albums or photos with your favourite service.',
+      },
+    ]}
+    img1={{ src: show0.src, alt: 'Showcase 1' }}
+    img2={{ src: show1.src, alt: 'Showcase 2' }}
+    img3={{ src: show2.src, alt: 'Showcase 3' }}
+    img4={{ src: show3.src, alt: 'Showcase 4' }}
+  />
+  <!-- Content Widget **************** -->
+
+  <Features4
+    isBeforeContent
+    isAfterContent
+    columns={3}
+    items={[
+      {
+        title: 'Open',
+        description:
+          'Lychee is completely <a href="https://github.com/LycheeOrg/Lychee">open-source</a>. Everyone can take advantage of the work we have already done and improve it. We are open for every suggestion or help.',
+        icon: 'tabler:brand-github',
+      },
+      {
+        title: 'Secure',
+        description:
+          'Your server, your data, your rules. Never depend on someone else. Lychee is a self-hosted solution, so you are in the full control of your photos.',
+        icon: 'tabler:lock',
+      },
+      {
+        title: 'Beautiful',
+        description:
+          'Our goal was to create a web app everyone can use. Lychee works intuitive and comes with a stunning, beautiful interface.',
+        icon: 'tabler:heart',
+      },
+    ]}
+  />
+
+  <!-- 
+  <Content
+    isReversed
+    tagline="Inside template"
+    title="AstroWind's Blueprint: Fun Meets Functionality!"
+    items={[
+      {
+        title: 'Built on top of Astro 4.0',
+        description:
+          'Benefiting from the performance and developer-friendly features of this modern static site generator.',
+      },
+      {
+        title: 'Styled using Tailwind CSS',
+        description:
+          'Facilitating rapid design and consistent styling with this highly popular utility-first CSS framework.',
+      },
+      {
+        title: 'Cross-browser compatibility',
+        description:
+          'Ensure your website looks and functions consistently across various web browsers, delivering a seamless experience to all users.',
+      },
+    ]}
+    image={{
+      src: 'https://images.unsplash.com/photo-1519389950473-47ba0277781c?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2070&q=80',
+      alt: 'Colorful Image',
+    }}
+  >
+    <Fragment slot="content">
+      <h3 class="text-2xl font-bold tracking-tight dark:text-white sm:text-3xl mb-2">Building on modern foundations</h3>
+      Gain a competitive advantage by incorporating industry leading practices
+    </Fragment>
+
+    <Fragment slot="bg">
+      <div class="absolute inset-0 bg-blue-50 dark:bg-transparent"></div>
+    </Fragment>
+  </Content> -->
```

Comment:
Any need for this?
```suggestion
```

Reply:
Not really, it is more boiler plate comment for styling later.
but can nuke :)